tfgrid3/umbrel/scripts/ufw.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -x
+
+ufw default deny incoming
```

Comment:
Not sure if it is a good idea to restrict this on behalf of the user as we are providing a standard umbrel server that allows the user to manage everything from UI without the need to SSHing to update Iptables rules. I have concerns regarding if this could interfere with the ability to install new services from the integrated app store.

Reply:
As I know the docker will manage the iptables for ports it uses, but I think it's better to make the machine secure as possible as Umbrel may have security issues with the insecure network please check this [link](https://github.com/getumbrel/umbrel/issues/190#issuecomment-683626245), for the most basic we need to limit ssh, so the default behavior for UFW is to deny incoming, even if I remove the 5th line. 
so I'm waiting for your opinion about that leave it as it is now,  modify the roles, or remove  the UFW at all

Reply:
leave it as it is now unless it causes issues with users' newly installed services